cmd/kubeadm/app/cmd/config.go

```diff
@@ -96,8 +96,10 @@ func NewCmdConfigPrint(out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "print",
 		Short: "Print configuration",
-		Long:  "This command prints configurations for subcommands provided.",
-		RunE:  cmdutil.SubCmdRunE("print"),
+		Long: dedent.Dedent(`
+			This command prints configurations for subcommands provided.
+			For details, see: https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2`),
```

Comment:
this is technically a quick workaround for not having something like `kubectl explain pod.spec` in kubeadm.
/assign @fabriziopandini